apps/api/src/app/invites/invites.controller.ts

```diff
@@ -30,17 +33,36 @@ export class InvitesController {
     @Post()
     @UseGuards(AuthGuard)
     @UseGuards(ThrottlerGuard)
```

Comment:
I think we can comment this line for now because it's not necessary to add limits. Maybe it will be necessary in the future. What do you think?

More info: https://docs.nestjs.com/security/rate-limiting

Reply:
Yes, probably not so great for hackathons / small projects right now. I'll comment this out.